src/editor_transport.rs

```diff
@@ -73,12 +74,19 @@ pub fn start(session: &str, initial_request: Option<String>) -> Result<EditorTra
                         let client = response.meta.client.as_ref();
                         let command = match client.filter(|&s| !s.is_empty()) {
                             Some(client) => {
-                                format!("eval -client {} -verbatim -- {}", client, response.command)
+                                let command = format!(
+                                    "eval -client {} -verbatim -- {}",
+                                    client, response.command
+                                );
+                                debug!("To editor `{}`: {}", response.meta.session, command);
+                                Cow::from(command)
+                            }
+                            None => {
+                                debug!("To editor `{}`: {}", response.meta.session, response.command);
+                                response.command
                             }
-                            None => response.command.to_string(),
                         };
 
-                        debug!("To editor `{}`: {}", response.meta.session, command);
```

Comment:
I had to move and duplicate debug! call in above match in order to preserve `Cow::borrowed` variant in None case.